src/styles/global/_base.scss

```diff
@@ -42,6 +42,7 @@ body {
     min-width: $buddylist-width;
     position: absolute;
     width: 100%;
+    left: 0;
```

Comment:
When in fullscreen everything is pushed to the left and incorrectly displayed. You can test this simply by double clicking on the spreed.me icon in the upper right hand corner.